### DISCUSSION TITLE
Fullscreen Mode: how to make it working at startup

### DISCUSSION THREAD

**Discussion by niccolov** [photobooth-app/photobooth-app#258] [Q&A] [answered]
Hi,

Finally everything is working 100%... I'm very happy and congrats!!!!

There is only one thing that I would like to implement: at boot chromium start automatically but it is not in fullscreen mode.

Following the instruction I hve:

~/Desktop/photobooth-app.desktop

[Desktop Entry]
Version=1.3
Terminal=false
Type=Application
Name=Photobooth-App
Exec=chromium-browser --kiosk --disable-features=Translate --noerrdialogs --disable-infobars --no-first-run --ozone-platform=wayland --enable-features=OverlayScrollbar --start-maximized http://localhost:8000/
StartupNotify=false

~/.config/wayfire.ini

[autostart]
chromium = chromium-browser --kiosk--disable-features=Translate --noerrdialogs --disable-infobars --no-first-run --ozone-platform=wayland --enable-features=OverlayScrollbar --start-maximized http://localhost:8000/

There is a difference on the files:

Exec=chromium-browser --kiosk --disable-features=Translate
VS
chromium = chromium-browser --kiosk--disable-features=Translate

Is it ok?

The problem is that at boot chromium start automatically but it is not in fullscreen mode.

Here an example:

![20240515_180137](https://github.com/photobooth-app/photobooth-app/assets/27413767/4d933a8b-33b3-4844-a8df-0c847628628e)

As you can see, it is not in fullscreen mode :(

What can I do/change to make it working in fullscreen mode?

The command "chromium-browser --start-fullscreen" could be a solution?

Thanks,

**Comment by mgineer85** [comment]
I think there should be a space after kiosk. 
change it to whatever works for you. The installation is just a guide and might have errors or the underlying system changes and needs to be updated.

**Reply by mgineer85** [reply]
Error fixed in docs https://github.com/photobooth-app/photobooth-docs/commit/f6a481359ba9f08ff56be71e8c13eca9967cf636

**Reply by baumrasen** [reply, marked as answer]
I think, there is also this error: [installation.md line 288](https://github.com/photobooth-app/photobooth-docs/blob/f6a481359ba9f08ff56be71e8c13eca9967cf636/docs/setup/installation.md?plain=1#L288)

**Reply by mgineer85** [reply]
Nice catch, update pipeline is running.
